test/suite/testInvPolyArcLength3.m

```diff
@@ -0,0 +1,43 @@
+% The PaCER Toolbox: testInvPolyArcLength3.m
+%
+% Purpose:
+%     - test the invPolyArcLength3 function
+%
+% Author:
+%     - Loic Marx, May 2019
+
```

Comment:
Is there a need here for the global variables?

Reply:
There is no need for the global variables here while reference data has been hard-coded

Reply:
Ok great! Maybe add a note.